examples/resnet/user_script.py

```diff
@@ -103,41 +101,8 @@ def resnet_calibration_reader(data_dir, batch_size=16):
 
 
 # keep this to demo/test custom evaluation function
-def eval_accuracy(model: OliveModel, data_dir, batch_size, device, execution_providers):
-    sess = model.prepare_session(inference_settings=None, device=device, execution_providers=execution_providers)
-    dataloader = create_dataloader(data_dir, batch_size)
-
-    preds = []
-    target = []
-    if model.framework == Framework.ONNX:
-        input_names = [i.name for i in sess.get_inputs()]
-        output_names = [o.name for o in sess.get_outputs()]
-        for input_data, labels in dataloader:
-            if isinstance(input_data, dict):
-                input_dict = {k: input_data[k].tolist() for k in input_data.keys()}
-            else:
-                input_data = input_data.tolist()
-                input_dict = dict(zip(input_names, [input_data]))
-            res = sess.run(input_feed=input_dict, output_names=None)
-            if len(output_names) == 1:
-                result = torch.Tensor(res[0])
-            else:
-                result = torch.Tensor(res)
-            outputs = post_process(result)
-            preds.extend(outputs.tolist())
-            target.extend(labels.data.tolist())
-
-    elif model.framework == Framework.PYTORCH:
-        for input_data, labels in dataloader:
-            if isinstance(input_data, dict):
-                result = sess(**input_data)
-            else:
-                result = sess(input_data)
-            outputs = post_process(result)
-            preds.extend(outputs.tolist())
-            target.extend(labels.data.tolist())
-
-    return AccuracyScore().measure(preds, target)
```

Comment:
we need keep these logic since user would have different logic generate the preds.

Reply:
Yeah this is what I mentioned in my PR description. I'm not sure what is the other use case for user for their own eval func. I only expose loaded model for the custom eval func now. I doubt if they insist to inference by themselves. If there is any different case from our inference.